Source/Urho3D/Utility/GLTFImporter.cpp

```diff
@@ -3568,10 +3568,13 @@ class GLTFSceneImporter : public NonCopyable
         staticModel.SetModel(model);
         staticModel.SetCastShadows(true);
 
-        const StringVector& materialList = modelImporter_.GetModelMaterials(meshIndex, skinIndex);
-        for (unsigned i = 0; i < materialList.size(); ++i)
+        const ResourceRefList& materialList = modelImporter_.GetModelMaterials(meshIndex, skinIndex);
+        if (materialList.type_ != Material::GetTypeStatic())
```

Comment:
To be honest - most of the Urho3D code doesn't check for the type. I guess it is nice to do anyway